utils/utils_supported.go

```diff
@@ -18,6 +18,10 @@ func RunUnderSystemdScope(pid int, slice string, unitName string) error {
 	properties = append(properties, newProp("PIDs", []uint32{uint32(pid)}))
 	properties = append(properties, newProp("Delegate", true))
 	properties = append(properties, newProp("DefaultDependencies", false))
+	if zeroTimeout {
+		var timeout uint64
+		properties = append(properties, newProp("TimeoutStopUSec", &timeout))
```

Comment:
I'd the opposite here: set `TimeoutStopUSec` to `math.MaxUint64` and also add `newProp("KillSignal", syscall.SIGUSR1))` that is used internally by conmon and it won't be forwarded to the container process.

In this way we won't lose the retcode from the container.

Reply:
Oooh. I like that. Will do.